services/core-api/Dockerfile

```diff
@@ -1,18 +1,28 @@
-FROM python:3.6
+FROM bitnami/python:3.6
```

Comment:
does this change the provider for the container image? if so, why?

Reply:
Good catch - this was from an attempt at fixing permissions issues in the container. Bitnami as a provider tends to be better with setting their permissions. Ultimately I fixed this by using https://github.com/bcgov/mds/pull/1942/files#diff-ae943a242e11cae5e334eb2a09b112dd40418caea9454ef04a2dda6aac807740R88 so I've reverted the change back to the old python:3.6